css-fonts-5/Overview.bs

```diff
@@ -180,6 +180,15 @@ The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors d
 <a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively. The `size-adjust`
 descriptor defines a scalar by which glyph outlines and metrics from this font are multiplied.
 
+The first value to 'ascent-override', 'descent-override', or 'line-gap-override' is used for
+vertical distance with upright glyph orientation (e.g. <a>horizontal writing modes</a> or
+<a>vertical writing modes</a> with <code>text-orientation: upright</code>) and horizontal
```

Comment:
In vertical writing modes with `text-orientation: upright`, should the first value be used at all? I guess I assumed that in this case what we refer to as "ascent" etc are the font's vertical-layout metrics, and the override for these is the second value, so the first is simply not relevant.

Reply:
Well, with `text-orientation: upright` in vertical writing modes, the glyphs have to be placed vertically somehow... So both values would end up being used. One to place the glyphs one-after-another, and the other to determine horizontal size that affects the line-to-line distance.

Reply:
Placing the vertical/upright glyphs one after another should normally be based on glyph advances, not on font-wide metrics. It's true that we need a fallback behavior for cases where the font has no vertical metrics (is that explicitly spec'd? need to check...), but I'm not sure the same overrides as are used for normal line-height in horizontal mode should also be used as a form of advance-override in vertical/upright.

Reply:
I think you're right that a better way of saying this would be "these metrics measure distance in the block direction". We'd use vertical advances in the inline direction.